vault/logical_system_paths.go

```diff
@@ -1044,6 +1162,8 @@ func (b *SystemBackend) internalPaths() []*framework.Path {
 				logical.ReadOperation: &framework.PathOperation{
 					Callback: b.pathInternalCountersRequests,
 					Summary:  "Backwards compatibility is not guaranteed for this API",
+					// callback only returns errors
```

Comment:
should we mark this endpoint as deprecated?

Reply:
> should we mark this endpoint as deprecated?

Yes, I think so. I believe there is a `Deprecated` field at the path level.

Reply:
https://github.com/hashicorp/vault/pull/19359